### ISSUE TITLE
Handling padding 0

### ISSUE THREAD

**Issue by marisancans** [kazuto1011/circular-conv-pytorch#1]
Hi, i'm trying to use a pretrained model and replace all Conv2D with CircularConv2d. How should I handle cases where padding for Conv2D is 0? Setting cpad=(0, 0, 0, 0) to CircularConv2d crashes the code. Thanks

**Comment by marisancans** [comment]
Nevermind, I had to leave places with 0 padding with the original Conv2D
